server/terraform/modules/myhealth/main.tf

```
@@ -82,6 +82,15 @@ resource "google_app_engine_application" "gae" {
   #}
 }
 
+# Logging storage
+# The _Default logging bucket is created automatically and set to 30 
+# days retention; this resource ensures it is kept at that length.
+resource "google_logging_project_bucket_config" "default" {
+    project    = var.project_id # google_project.project.name
+    location  = "global"
```

WHO Policy is to use Switzerland wherever possible, so it should inherit the same location as used throughout the rest of the project:
```suggestion
    location  = var.region
```

---

This is not an option for the _Default bucket. It does look like this is possible with additional configuration: https://cloud.google.com/logging/docs/regionalized-logs

---

@matthewblain - use `europe-west1` (Belgium) for now. We'll switch to Switzerland once it's available.